teamscale-gradle-plugin/src/main/kotlin/com/teamscale/GitRepositoryHelper.kt

```diff
@@ -14,13 +14,14 @@ object GitRepositoryHelper {
 
     /** Returns the last commit in the given git repository. */
     @Throws(IOException::class)
-    fun getHeadCommitDescriptor(baseDirectory: File): CommitDescriptor {
+    fun getHeadCommitDescriptor(baseDirectory: File): Pair<CommitDescriptor, String> {
         val git = Git.open(baseDirectory)
         val repository = git.repository
         val branch = repository.branch
         val commit = getCommit(repository, branch)
         val time = commit.commitTime.toLong()
```

Comment:
are these seconds (due to the * 1000 below)? seems not obvious from identifiers here

Reply:
We are interacting with the jgit API here, which uses seconds because that is also the resolution Git uses internally. Made it a bit clearer by renaming the variable to `commitTimeSeconds`. So we are not losing precision, but adding "artificial precision"